source/_base.css

```diff
@@ -7,6 +7,18 @@
   animation-iteration-count: infinite;
 }
 
+.animated.loop-1 {
+  animation-iteration-count: var(--animate-loop);
+}
+
+.animated.loop-2 {
+  animation-iteration-count: calc(var(--animate-loop) * 2);
+}
+
+.animated.loop-3 {
+  animation-iteration-count: calc(var(--animate-loop) * 3);
+}
```

Comment:
I think we need an `infinite` class too

Reply:
@eltonmesquita agree! let me update this... thank you! I miss that one...

Reply:
I can't update it @eltonmesquita I deleted my forked repo for this and I can't get the updates I did here. I will update it if this PR has been merge, or is there anyway to get the updates here.

Reply:
Just clone this repo, then:
`git checkout -b WarenGonzaga-dev dev` and

`curl -L https://github.com/daneden/animate.css/pull/1029.patch | git am -3` that should do it.

Reply:
Thanks @eltonmesquita! this is what I'm looking for

Reply:
@eltonmesquita we have already infinite class... what do you mean create it as variable?

Reply:
Oh, right. I forgot about that. Now we have a consistency problem.

When repeating animations, you can use the `loop` and `infinite` classes, two different "syntaxes". I don't know if I'm over-worried but what do you think?

Reply:
@eltonmesquita let's choose between infinite class or loop-infinite class? which is more way better?  for me... infinite alone is more useful... although they are aware on loop classes.